ansible/bambimoloch.yml

```diff
@@ -0,0 +1,21 @@
+---
+  - hosts: all
+    become: yes
+    become_method: sudo
+    vars_files:
+      - config_bambi.yml
+      - enomoloch-get-pcaps
+    roles:
+        - bambi-ssh-keys
+        - firewall
+        - wireguard
+        - enomoloch
+        - role: filebeat
+          vars:
+            elk: 192.168.1.0
```

Comment:
Can't we use the domain name here?

Reply:
No we can't, the domain names are for the public ipv4s

Reply:
That's also more fitting for #4 and was replaced with a variable there